internal/cmd/internal/olmv1/printing.go

```diff
@@ -0,0 +1,58 @@
+package olmv1
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/duration"
+
+	catalogdv1 "github.com/operator-framework/catalogd/api/v1"
+	olmv1 "github.com/operator-framework/operator-controller/api/v1"
+)
+
+func printFormattedOperators(extensions ...olmv1.ClusterExtension) {
+	tw := tabwriter.NewWriter(os.Stdout, 3, 4, 2, ' ', 0)
+	_, _ = fmt.Fprint(tw, "NAME\tINSTALLED BUNDLE\tVERSION\tSOURCE TYPE\tINSTALLED\tPROGRESSING\tAGE\n")
+	for _, ext := range extensions {
+		age := time.Since(ext.CreationTimestamp.Time)
+		_, _ = fmt.Fprintf(tw, "%s\t%s\t%s\t%s\t%s\t%s\t%s\n",
+			ext.Name,
+			ext.Status.Install.Bundle.Name,
+			ext.Status.Install.Bundle.Version,
+			ext.Spec.Source.SourceType,
+			status(ext.Status.Conditions, olmv1.TypeInstalled),
```

Comment:
when you get to unit tests at some point, probably should have one (+?)  showing we don't list olmv0 resources. I think `kubectl-operator` doesn't know what might be on a cluster, could have v0 and v1 operators installed, certainly catalog can have both.

Reply:
does `olmv0` use the same api version and kind as `olmv1` - ie. `olm.operatorframework.io/v1` and `ClusterExtension`?

Reply:
yeah we did make the break to new resource kinds, so I guess it's safe

Reply:
> I think kubectl-operator doesn't know what might be on a cluster, could have v0 and v1 operators installed, certainly catalog can have both.

But it should not try to detect and change its operating mode automatically.

At the risk of being too opinionated, let me try to be unequivocal. As soon as sufficient functionality exists in kubectl-operator to support OLMv1, we should either remove OLMv0 support or put it somewhere that requires a user to specifically decide to use it.